whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -155,16 +159,19 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
                 case DOMAIN -> {
                     final Domain domain = (Domain) getRdapObject(requestUrl, object, null);
                     mapRedactions(domain);
+                    rdapRelationService.mapRelationLinks(domain, requestUrl);
```

Comment:
If this needs to be  added for every case can it be outside switch ?

Reply:
yes, I have done this already as part of my last review/refactor. Thanks

Reply:
anyways we are already checking the Response type inside mapCommonRelationLinks so it should be safe.
I am refactoring it

Reply:
I did refactor it, this can go inside getRdapObject